### DISCUSSION TITLE
How to delpoy Frontend and backend at the same time

### DISCUSSION THREAD

**Discussion by highfly117** [vercel/community#2696] [Help]
### Summary

Hi 

I have an App I'm building (just a demo for employers). Currently, it's a react front end of a folder called client and an express backend call API. is there any way to deploy both at the same time?

Thanks
Adam

### Example

https://github.com/highfly117/MyWork

### Steps to Reproduce

deployed the app but best i can manage is build the client by setting the root folder to the client folder

**Comment by amyegan** [comment]
Hi @highfly117! It looks like you have a sort of monorepo setup. I think our docs on [using monorepos with Vercel](https://vercel.com/docs/concepts/monorepos) are a good place to start. Once you have set that up, both projects will automatically deploy every time a change is pushed to the repo.

You can also use the [Ignored Build Step](https://vercel.com/guides/how-do-i-use-the-ignored-build-step-field-on-vercel) setting to skip the build for the client if only the api was updated, or vice versa. You can find a nice explanation of that setup at https://dev.to/gdbroman/how-to-trigger-fewer-vercel-deploys-for-your-monorepo-4ok1
